nss-day-cohort-47/reactive-nutshell-the-fantastic-404s
# Lm starting css and public message

# Description
- Starting basic CSS
- Adding Footer
- Coloring header (logo not working yet)
- Making public messages edit/delete user-specific (delete still deletes all messages)
- Name of sender on public messages

Fixes # (issue): making edit/delete buttons appear for their own public messages

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Please review dashboard to make sure you can see edit/delete for your public messages (edit should work, delete will not)

# Checklist:
- [x] I have added test instructions that prove my fix is effective or that my feature w
